conduct/COC_POLICY.md

```diff
@@ -0,0 +1,109 @@
+This document defines the OpenJS Foundation's code of conduct policy.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Policy
+
+The OpenJS Foundation (OpenJSF) delegates to the Cross Project Council (CPC), via its [charter][], the responsibility of choosing, maintaining, and enforcing a Code of Conduct (CoC) for all of its projects, collab spaces, and for any related activities.
+
+## Code of Conduct
+
+The OpenJSF's code of conduct is located at [code-of-conduct.openjsf.org][CoC]. It is based on the [Contributor Covenant](https://www.contributor-covenant.org/).
+
+A common code of conduct allows a common escalation and appeal process and allows better support of projects in their efforts to apply the code of conduct.
+
+All OpenJSF projects, collab spaces, and any activities related to the OpenJSF are operating under this code of conduct unless otherwise stated. (For example, events organized in partnership with other organizations might be operating under a different code of conduct.)
+
+Projects joining the foundation that already have a code of conduct will need to adopt the OpenJSF's CoC within three (3) months. Projects that don't already have a code of conduct operate under the OpenJSF's CoC as soon as they enter incubation.
+
+## Reporting incidents
+
+Reporting incidents should be as simple and flexible as possible.
+
+All information necessary to report an incident must be publicly documented directly alongside the code of conduct itself. This information must contain at least the following:
+
+1. A general email address at which incidents can be reported.
+2. The name and email address of each recipient of the general email address.
+
+Reporters may choose to report an incident by using the general email address provided or by contacting any of its recipients individually, either by using their provided email, reaching out in person, or by any other reasonable means.
+
+An acknowledgment of receipt should be promptly issued to incident reporters.
+
+The name and contact details of the people responsible for managing the incident report should be provided to reporters as soon as possible (to uncover conflicts of interest upfront and allow for escalation).
+
+## Incident Response
```

Comment:
Node.js over the years developed quite a good handbook for handling Moderation incidents (Moderation Policy): https://github.com/nodejs/admin/blob/main/Moderation-Policy.md

What about adopting it? It feels quite complete and very well written...

Reply:
@ovflowd: Could you please file a separate issue for your suggestion to adopt a uniform moderation approach across the foundation? This seems like a well-scoped and distinct enough issue that we can keep separated.

Reply:
@PaulaPaul: wrt to getting inspiration from the CNCF, my belief was that this was more about the CoC Team structure and its processes than about moderation. Looking forward to chat about it tomorrow, though.

Reply:
> Social media or other online behavior targeted at contributors in the context of their contributions to the Foundation?

As much as I'd love that, I wonder how we'd enforce CoC on platforms not governed by us?

Reply:
> @ovflowd: Could you please file a separate issue for your suggestion to adopt a uniform moderation approach across the foundation? This seems like a well-scoped and distinct enough issue that we can keep separated.

I may, but I don't think we can merge this PR as it is without actually providing these processes here; Since this PR effectively changes the moderation processes, and in a technical standpoint, it doesn't make sense to merge "tech debt"; I wouldn't merge this PR changes to moderation processes if we already have plans to update them in an uniform way. Wdyt?

Reply:
@PaulaPaul wrote:
> My thoughts there were not about enforcement but for the scope of the CoC. For example, if I am harassed online because of work I contributed to the foundation, that violation of the CoC would be in social media, not in a comment on a repo or in Slack. If that ever happened to me I would like to know if I could raise it as a violation of the CoC (one way or another). It's not realistic to think the CPC could police social media but could act as a place to report CoC violations of any kind specifically related to foundation contributions. Does that make sense?

This is mostly covered in the [scope section of the CoC](https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md#scope):

> _This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event._

Reply:
> Are we talking about incident report management here, or about moderation? Moderation (i.e. pro-actively managing comments, etc., before incidents happen and are reported) is managed by the projects themselves. My understanding is that the policy you linked is a moderation policy, not a process to manage incident reports. 

Yes and No. The Moderation process/policy of Node.js describes an overall guidelines how Moderators should handle incidents (either proactively or through reports);

And I believe moderation in-repository is fundamental here, too; We do, after all (CPC and the Foundation) live within our Slack and this GitHub organization.

Reply:
I've created an Issue: https://github.com/openjs-foundation/cross-project-council/issues/1215

Reply:
Can we mark this as resolved? @PaulaPaul if you believe the scope of the CoC should be changed, can you file a separate issue for that please?